src/main/java/net/javadiscord/javabot/data/config/guild/HelpConfig.java

```diff
@@ -47,79 +31,41 @@ public class HelpConfig extends GuildConfigItem {
 	 */
 	private long helpPingRoleId;
 
-	/**
-	 * The strategy to use when naming help channels. This is only used when
-	 * {@link HelpConfig#recycleChannels} is set to false.
-	 */
-	private String channelNamingStrategy = "animal";
-
-	/**
-	 * If true, the system will manage a fixed set of help channels which are
-	 * created in advance. If false, the system will create and remove channels
-	 * as needed to maintain the {@link HelpConfig#preferredOpenChannelCount}.
-	 * Note that if this is true, the preferred open channel count is ignored.
-	 */
-	private boolean recycleChannels = false;
-
-	/**
-	 * The string which is shown as the 'topic' for open channels.
-	 */
-	private String openChannelTopic = "Ask your question here!";
-
-	/**
-	 * The message that's sent in a recycled help channel to tell users that it
-	 * is now open for someone to ask a question.
-	 */
-	private String reopenedChannelMessage = "`✅` **This channel is now available!**\n> This channel is no longer reserved. Feel free to ask your question here!";
-
 	/**
 	 * The message that's sent as soon as a user asks a question in an open help
 	 * channel. This is only sent if it's not null.
 	 */
-	private String reservedChannelMessage = "`⌛` **This channel has been reserved for your question.**\n> Please use `/unreserve` when you\u0027re finished.";
+	private String reservedChannelMessageTemplate = "`⌛` **This post has been reserved for your question.**\n> Hey %s! Please use `/close` or the 'Close Post' button above when you're finished. Please remember to follow the help guidelines. This post will be automatically closed after %s minutes of inactivity.\n\n**TIP:** Narrow down your issue to __simple__ and __precise__ questions to maximize the chance that others will reply in here.";
```

Comment:
wdym?

Reply:
I meant
```
Please use `/close` or the `Close Post` button
``` instead of ```
Please use `/close` or the 'Close Post' button
``` (notice the backticks instead of single quotes)

Reply:
yeah I've used way too much dart recently, fixed that